src/components/ActionableList/ActionableList.tsx

```diff
@@ -33,9 +34,20 @@ export const ActionableList = <T,>({
           className={classNames(
             x.secondary && 'Layer__actionable-list-item--secondary',
             x.asLink && 'Layer__actionable-list-item--as-link',
+            selectedId === x.id && 'Layer__actionable-list__item--selected',
           )}
         >
-          <Text>{x.label}</Text>
+          <div className='Layer__actionable-list__content'>
+            <Text size={TextSize.sm}>{x.label}</Text>
+            {x.description && x.label !== 'See all categories' && (
```

Comment:
I would rather go with something more generic instead of checking specific text: `"See all categories"`. Maybe we can add additional prop to the option?